pyproject.toml

```diff
@@ -29,4 +29,4 @@ requires = ["setuptools>=61.0"]
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools]
-packages = ["browserbase"]
+packages = ["browserbase", "browserbase.helpers"]
```

Comment:
idt this is necessary unless you want people pip installing only the helpers

Reply:
I'm not sure to be honest.

Reply:
yeah this feels unnecessary